build.sbt

```diff
@@ -255,6 +255,7 @@ lazy val mtagsShared = project
     Compile / packageSrc / publishArtifact := true,
     libraryDependencies ++= List(
       "org.lz4" % "lz4-java" % "1.8.0",
+      "com.google.protobuf" % "protobuf-java" % "3.22.3",
```

Comment:
Why is it needed?

Reply:
It is required for `Compression` object which is used in `CompressedPackageIndex`. While I could have created e.g. an uncompressed package index to use in dotty, it just was more convenient to use the same one in both places, especially considering that `protobuf-java` is already a dependency in the compiler, and they guarantee binary compatibility between major releases.